src/styles.ts

```diff
@@ -0,0 +1,142 @@
+import styled, { createGlobalStyle } from "styled-components";
+import { HashLink } from "react-router-hash-link";
+import img from "./highlighter.png";
+
+export const GlobalStyle = createGlobalStyle`
+    body {
+        font-family: 'Josefin Sans', sans-serif;
+        font-weight: 300;
+        font-size: 18px;
+        };
+
+    h1 {
```

Comment:
actually this looks like a good idea to unify styles for some element on different browsers. Good job with that